scripts/periodics_pass_rates.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
```

Comment:
Since we're doing OCD, this file definitely contains bash-specific syntax like `$()` :slightly_smiling_face:

Reply:
It's not OCD; it literally didn't work on my machine :slightly_smiling_face:.  I'll change it to `#!/usr/bin/env bash`, would that be satisfactory to you?